CoverMyDotNet/Requests/SearchForm.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using RestSharp;
+
+namespace CoverMyDotNet.Requests
+{
+	public class SearchForm : RestRequest
```

Comment:
@twyfordr RestApi doesn't force you to do that, but I thought that it was the best way to go.